Library/Formula/pam_u2f.rb

```diff
@@ -0,0 +1,37 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
+  url "https://github.com/Yubico/pam-u2f.git", :commit => "95fb0ec65475e8afa15aef1c14d1457d1f80fece"
+  version "1.0.2"
+  sha256 "d77dd150d65836c7b9a12d75d22159484663c11cea5739653bf31c44a3609522"
+
+  head "https://github.com/Yubico/pam-u2f.git"
+
+  depends_on "libu2f-host" => :build
+  depends_on "libu2f-server" => :build
+
+  depends_on "pkg-config" => :build
+  depends_on "libtool" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "asciidoc" => :build
+
+  def install
+    system "autoreconf", "--install"
+
+    ENV["A2X"] = "#{Formula["asciidoc"].opt_bin}/a2x --no-xmllint"
+    args = ["--prefix=#{prefix}", "--with-pam-dir=#{lib}/pam"]
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  def caveats; <<-EOS.undent
+    To use the PAM module, specify the full path in your PAM configuration. E.g.
```

Comment:
Updated the post-installation instructions. Give them a look and let me know what you think.